libraries/AP_Relay/AP_Relay.cpp

```diff
@@ -25,10 +25,19 @@
 #elif CONFIG_HAL_BOARD == HAL_BOARD_VRBRAIN
 #define RELAY1_PIN_DEFAULT 33
 #define RELAY2_PIN_DEFAULT -1
+#elif CONFIG_HAL_BOARD == HAL_BOARD_LINUX
+#ifdef HAL_BOARD_SUBTYPE_LINUX_BLUE
+#define RELAY1_PIN_DEFAULT 57
+#define RELAY2_PIN_DEFAULT 49
+#define RELAY3_PIN_DEFAULT 116
+#define RELAY4_PIN_DEFAULT 113
+#endif
 #else
 // no relay for this board
 #define RELAY1_PIN_DEFAULT -1
 #define RELAY2_PIN_DEFAULT -1
+#define RELAY3_PIN_DEFAULT -1
```

Comment:
you need to add this to all boards ! otherwise it will be missing

Reply:
Better to remove the else and do a block of

```
#ifndef RELAY1_PIN_DEFAULT
  #define RELAY1_PIN_DEFAULT -1
#endif

[...]
```
Then all the above defining -1 can be removed.

Reply:
agree

Reply:
The above suggestion is now implemented and confirmed working on the Blue.  I'll cleanup the PR and push the changes later today...